crates/humanode-peer/src/cli/subcommand/bioauth/key/insert.rs

```diff
@@ -55,15 +55,14 @@ pub async fn ensure_bioauth_key_absent<PK: AppPublic>(
 /// A helper function to insert bioauth key into the keystore.
 pub async fn insert_bioauth_key<PK: AppPublic>(
     suri: &str,
-    keystore: Arc<dyn CryptoStore>,
+    keystore: Arc<dyn Keystore>,
 ) -> sc_cli::Result<()> {
     // We don't use a password for keystore at the current moment. That's why None is passed.
-    let pair = utils::pair_from_suri::<<PK as AppKey>::Pair>(suri, None)?;
+    let pair = utils::pair_from_suri::<<PK as AppCrypto>::Pair>(suri, None)?;
     let public = pair.public().as_ref().to_vec();
     keystore
```

Comment:
Do this with `tokio::block_in_place`

Reply:
Blocking I/O on reactor thread?

Reply:
Discussed: we need to verify that keystore operations are used with either tokio::block_in_place or spawn::blocking (prefer option).

Reply:
https://github.com/humanode-network/humanode/pull/861/commits/29f2fdc71c20bc3b23b3b76a0d92e9aff2434f04